packages/app-desktop/gui/NoteEditor/NoteBody/TinyMCE/TinyMCE.tsx

```diff
@@ -86,6 +86,21 @@ function createSyntheticClipboardEventWithoutHTML(): ClipboardEvent {
 	return new ClipboardEvent('paste', { clipboardData });
 }
 
+export enum TinyMceEditorEvents {
+	KeyUp = 'keyup',
+	KeyDown = 'keydown',
+	KeyPress = 'keypress',
+	Paste = 'paste',
+	PasteAsText = 'pasteAsText',
+	Copy = 'copy',
+	CompositionEnd = 'compositionend',
+	Cut = 'cut',
+	JoplinChange = 'joplinChange',
+	Undo = 'Undo',
+	Redo = 'Redo',
+	ExecCommand = 'ExecCommand',
+}
```

Comment:
Thanks for creating the type, but since it's shared by multiple files, could you move it under /utils/types.ts please? This is to avoid circular dependencies which can make TS randomly fail to compile

Reply:
Thanks for the suggestion, I just did that.